lms/djangoapps/verify_student/views.py

```diff
@@ -946,6 +947,27 @@ def _send_email(user_id, subject, message):
     user.email_user(subject, message, from_address)
 
 
+def _set_requirement_status(attempt, namespace, status, reason=None):
+    """Set requirement status and the reason of status for given user in
+    attempt.
+    """
+    checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
+    if checkpoints:
+        course_key = checkpoints[0].course_id
```

Comment:
Is it safe to assume that the first `checkpoint` will always be the one required here?